tools/crm_resource.c

```diff
@@ -582,6 +567,23 @@ static GOptionEntry addl_entries[] = {
     { "interval", 'I', G_OPTION_FLAG_NONE, G_OPTION_ARG_STRING, &options.interval_spec,
       "Interval of operation to clear (default 0) (with -C -r -n)",
       "N" },
+    { "class", 0, G_OPTION_FLAG_NONE, G_OPTION_ARG_CALLBACK, class_cb,
+      "The standard the resource agent confirms to (for example, ocf).\n"
```

Comment:
This is probably supposed to be "conforms to".  It was that way before.  I imagine I just typo'd it when rearranging this code.

Reply:
Good catch